rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -444,10 +404,12 @@ public String getTaskServerLogsByTag(String jobId, String tag)
     @Override
     public boolean isConnected() {
         boolean isConnected = false;
-        try {
-            isConnected = restApi().isConnected(sid);
-        } catch (NotConnectedRestException e) {
-            // ignore
+        if (initialized) {
+            try {
+                isConnected = restApi().isConnected(sid);
+            } catch (NotConnectedRestException e) {
```

Comment:
If you want to be sure that `isConnected` never throw an exception (as stated in your commit message), then `Exception` or even `Throwable` should be catched (i.e. an IOException could be thrown).